app-modules/campaign/src/Filament/Blocks/EngagementBatchEmailBlock.php

```diff
@@ -73,7 +74,25 @@ public function generateFields(string $fieldPrefix = ''): array
             TextInput::make($fieldPrefix . 'subject')
                 ->columnSpanFull()
                 ->placeholder(__('Subject'))
-                ->required(),
+                ->required()
+                ->visible(! RefactorEngagementCampaignSubjectToJsonb::active()),
+            TiptapEditor::make($fieldPrefix . 'subject')
+                ->recordAttribute('data.subject')
+                ->label('Subject')
+                ->mergeTags($mergeTags = [
+                    'student first name',
+                    'student last name',
+                    'student full name',
+                    'student email',
+                    'student preferred name',
```

Comment:
Due to a recent change in one of Dan's ticket, this should now be `recipient` instead of `student`.